templates/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1>Login</h1>
+<form method="post">
```

Comment:
После тут нужно добавить следящее
```suggestion
<h1>Login</h1>
{% with messages = get_flashed_messages() %}
{% if messages %}
    {% for message in messages %}
        {{ message }}
    {% endfor %}
{% endif %}
{% endwith %}
<form method="post">
```